.github/release-notification.yml

```diff
@@ -0,0 +1,19 @@
+name: Release Notification
+
+on:
+  release:
+    types: [released]

```

Comment:
This will not work for Typescript, here no release is created on github. Rather this is a mono repo that contains multiple packages. You might need to automate this action via some other trigger @MaryamAdnan3 might help here^

Reply:
Yes we don't initiate release via github action. At the moment its a manual process to release typescript monorepos.